jax/numpy/lax_numpy.py

```diff
@@ -644,7 +644,10 @@ def _reduction_init_val(a, init_val):
 
 
 @_wraps(onp.mean)
-def mean(a, axis=None, dtype=None, keepdims=False):
+def mean(a, axis=None, dtype=None, out=None, keepdims=False):
+  if out is not None:
+    raise ValueError("mean does not support `out` argument.")
```

Comment:
argument*s* ?

Same everywhere.

Reply:
I think it makes more sense to be singular since `out` is a single argument.

Reply:
I misunderstood your point! I'll add a "the" to make the sentence parse.